MAUI/LinearGauge/pointers.md

```diff
@@ -0,0 +1,186 @@
+description: Learn here all about the interactions in Syncfusion .NET MAUI Linear Gauge (SfLinearGauge) control and more
+platform: maui
+control: SfLinearGauge
+documentation: ug
+---
+
+# Pointers in .NET MAUI Linear Gauge (SfLinearGauge)
+
+The pointer is used to indicate values on a scale. The Linear Gauge control has three types of pointers:
+
+[`Bar pointer`]()
+[`Shape Pointer`]()
+[`Content pointer`]()
+
+All the pointers can be customized as needed. You can add multiple pointers to the gauge to point multiple values on the same scale. The value of the pointer is set using the Value property.
+
+![Pointer in linear gauge](images/pointers/pointers.PNG)
+
+## Interaction with pointers
+
+The following code sample demonstrates how to update simple marker pointer value based on swipe or drag gestures.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<gauge:SfLinearGauge >
+                <gauge:SfLinearGauge.MarkerPointers>
+                    <gauge:LinearShapePointer Value="70" IsInteractive="True" Fill="Blue"/>
+                </gauge:SfLinearGauge.MarkerPointers>
+            </gauge:SfLinearGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearGauge gauge = new SfLinearGauge();
+		gauge.MarkerPointers.Add(new LinearShapePointer()
+		{
+			Value = 70,
+			IsInteractive = true,
+			Fill=new SolidColorBrush(Colors.Blue)	
+		});
+		this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Simple pointer interaction in linear gauge](images/pointers/simple_interaction.gif)
+
+## Step frequency
+
+The [`StepFrequency`]() property is used to specify the interval between snap points while dragging the pointer.
+
+For example, if the value of [`StepFrequency`]() is 20, the pointer will not move continuously while dragging; instead, it will update in terms of 20.
+
+N> To work with the [`StepFrequency`]() value, enable pointer interaction support.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<gauge:SfLinearGauge>
+                <gauge:SfLinearGauge.MarkerPointers>
+                    <gauge:LinearShapePointer Value="70" IsInteractive="True" 
+                                              StepFrequency="5" Fill="Blue"/>
+                </gauge:SfLinearGauge.MarkerPointers>
+            </gauge:SfLinearGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfLinearGauge gauge = new SfLinearGauge();
+		gauge.MarkerPointers.Add(new LinearShapePointer()
+		{
+			Value = 70,
+			IsInteractive = true,
+			StepFrequency = 5,
+			Fill=new SolidColorBrush(Colors.Blue)	
+		});
+		this.Content = gauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Drag offset
+
+The [`DragOffset`]() property is used to specify the outer dragging offset for pointer. When the pointer is pressed close to the pointer, this value helps in dragging the pointer. The default value of [`DragOffset`]() is 15d.
+
+For example, if the value of [`DragOffset`]() is 20, the pointer will be dragged while interact with pointer around 20 pixel. 
```

Comment:
if the value of [`DragOffset`]() is 20, the pointer will be dragged while interacting with another pointer around 20 pixels.